<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LinkLibrariesPage</class>
 <widget class="QWidget" name="LinkLibrariesPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>653</width>
    <height>488</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="lbl_image">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="../res/TemplateFramework.qrc">:/Framework/pageImage</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QGroupBox" name="gbox_linkLibraries">
       <property name="title">
        <string>Link to which libraries:</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QCheckBox" name="cbx_foundationLib">
            <property name="text">
             <string>Foundation</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="cbx_openMayaLib">
            <property name="text">
             <string>OpenMaya</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="cbx_openMayaUILib">
            <property name="text">
             <string>OpenMayaUI</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="cbx_openMayaAnimLib">
            <property name="text">
             <string>OpenMayaAnim</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="cbx_openMayaFXLib">
            <property name="text">
             <string>OpenMayaFX</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="cbx_openMayaRenderLib">
            <property name="text">
             <string>OpenMayaRender</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="cbx_imageLib">
            <property name="text">
             <string>Image</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="cbx_mocapLib">
            <property name="text">
             <string>libMocap</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../res/TemplateFramework.qrc"/>
 </resources>
 <connections/>
</ui>
